README.md

```diff
@@ -99,27 +99,36 @@ Here are the default block classes that can be extended:
 
 | Class Name | Noteworthy Behavior |
 | --- | --- |
-| BookmarkBlock | |
+| BookmarkBlock | Converts visual bookmark into plain text link in markdown, using the caption as the link text. |
+| BreadcrumbBlock | Ignore |
 | BulletedListItemBlock | |
 | CalloutBlock | The content of the callout block is extracted, but the emoji and background color are ignored. |
-| ChildPageBlock | |
-| FencedCodeBlock | |
+| ChildDatabaseBlock | Converts database to markdown in different file in directory and writes a relative link to the markdown|
```

Comment:
The difficulty with exporting the child database to markdown is that there's no way to represent a database as a single markdown file. Depending on the selected output format, a database is either a single YAML file or many markdown files.

For now, I propose we just ignore child databases.